web/components/quests-or-streak.tsx

```diff
@@ -190,8 +194,10 @@ const QuestRow = (props: {
   status: string
   reward: number
   info?: string
+  link?: string
```

Comment:
Prefer the name `href` over `link` (it is not a full link)

Reply:
Renamed, thanks!